examples/box2d/acrobot_dualdqn.py

```diff
@@ -56,8 +57,10 @@ def test_dqn(args=get_args()):
     train_envs.seed(args.seed)
     test_envs.seed(args.seed)
     # model
-    net = Net(args.layer_num, args.state_shape,
-              args.action_shape, args.device, dueling=(2, 2)).to(args.device)
+    dueling = ({"hidden_layer_size": [128, 128]},
+               {"hidden_layer_size": [128, 128]})
```

Comment:
Tuple[List[int], List[int]] or Dict[str, List[int]] instead of Tuple[Dict[str, List[int]]]

e.g., 
```
dueling = ([128, 128], [128, 128])
# or
dueling = {"hidden_layer_size_q": [128, 128], "hidden_layer_size_v": [128, 128]}
```